[OPBGUGS-50844] Add steps to enable FIPS mode on IBM Z with ABI (#88558)
openshift/openshift-docs: modules/agent-installer-configuring-fips-compliance.adoc

During a cluster deployment, the Federal Information Processing Standards (FIPS) change is applied when the Red Hat Enterprise Linux CoreOS (RHCOS) machines are deployed in your cluster. For Red Hat Enterprise Linux (RHEL) machines, you must enable FIPS mode when you install the operating system on the machines that you plan to use as worker machines.

[IMPORTANT]
====

> {product-title} requires the use of a FIPS-capable installation binary to install a cluster in FIPS mode.

Reviewer: ```suggestion
You must use a FIPS-capable installation binary for {product-title} to install a cluster in FIPS mode.
```

- Consider rephrasing in a more user-focused approach -- something like the above suggested change...

  name: sno-cluster
fips: True
----
+
[IMPORTANT]
====

> If you want to enable FIPS mode on {ibm-z-name} clusters, you must also enable FIPS in either the `.parm` file or using `virt-install` as outlined in the procedures for manually adding {ibm-z-name} agents.

Reviewer 2: Suggestion for simplicity, up to you
```suggestion
To enable FIPS mode on {ibm-z-name} clusters, you must also enable FIPS in either the `.parm` file or using `virt-install` as outlined in the procedures for manually adding {ibm-z-name} agents.
```
